src/libs/Logger.js

```diff
@@ -46,19 +43,43 @@ module.exports = class Logger {
     return ErrorNoStack;
   }
 
-  buildLog (logLevelindex, event, data = null, meta = null) {
-    if (data instanceof Error) {
-      let error = data;
+  serializeError (logLevelindex, logData) {
+    let serialized;
+    const isError = logData instanceof Error;
+    let baseError;
+
+    if (isError) {
+      baseError = logData;
+    } else if ((logData || {}).error instanceof Error) {
+      baseError = logData.error;
+    }
+
+    if (baseError) {
       if (logLevelindex < this.stackLevelIndex) {
         // Remove stack when not required
-        error = this.removeStack(error);
+        baseError = this.removeStack(baseError);
       }
-      data = {
-        error: pino.stdSerializers.err(error)
-      };
+      serialized = isError
+        ? { error: pino.stdSerializers.err(baseError) }
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Convert this usage of the ternary operator to an "if"/"else" structure. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)